fedWireMessage.go

```diff
@@ -1502,3 +1513,18 @@ func (fwm *FEDWireMessage) isRemittanceValid() error {
 	}
 	return nil
 }
+
+func (fwm *FEDWireMessage) getValidation() *ValidateOpts {
```

Comment:
Is this used anywhere?

Reply:
we need to implemented get/set validation opts in file side too
the reason is that validate function implemented as public function in file side